packages/ember-auto-import/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "include": ["./ts/**/*.ts"],
   "compilerOptions": {
+    "lib": ["es7"],
```

Comment:
Was required in order to allow usage of `Array#includes`, which is supported down to Node 6.4: https://node.green/#ES2016-features-Array-prototype-includes

`Array#includes` is already used in this project, but on an `any` type, which is why `tsc` has not complained yet.